.idea/k8s@home charts.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Might need to `.gitignore` this, as not everyone will be using a Jetbrains IDE.

Reply:
Good catch! Didnt even notice.